components/cart/checkout/CheckoutLogin.vue

```diff
@@ -39,9 +39,12 @@ export default defineNuxtComponent({
     })
     watch(
       () => user?.value,
-      (value) => {
-        if (value instanceof User) {
-          success()
+      (newValue, oldValue) => {
+        if (newValue instanceof User) {
+          if (oldValue == null) {
```

Comment:
@gervye could we merge both conditions into the same if statement?

Reply:
Yes, the merge is now done